services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = User(username=username, password=password)
```

Comment:
Avoid setting the password directly on the `User` object. Use `set_password` to ensure the password is hashed before saving.